clang-tools-extra/docs/ReleaseNotes.rst

```diff
@@ -351,6 +351,10 @@ Changes in existing checks
   <clang-tidy/checks/modernize/use-starts-ends-with>` check to also handle
   calls to ``compare`` method.
 
+- Improved :doc:`modernize-use-std-format` and
```

Comment:
remove modernize-use-std-format from here, as it's new check, only leave std-print

Reply:
I spotted that just too late...